src/Home.tsx

```diff
@@ -44,6 +44,9 @@ export const Home: React.FC = () => {
   const [accountAddress, setAccountAddress] = React.useState<
     null | string | undefined
   >(account ? account : null);
+  const [underlyingTokens, setUnderlyingTokens] = React.useState<
```

Comment:
Instead of this we need to have protocol wise tokens which can have array of underlying tokens.
Ex: 
1) Uniswap -> UNI-V2 USDC/DAI pair will have USDC and DAI as underlying tokens

So, `protocols: Array<Protocol>`, `protocol: Array<ProtocolTokenBalance>`, `protocolTokenBalance: Array<TokenBalance>`, `protocolTokenBalance` would contain value and price with other basic token info(name, symbol, etc).

We will have this protocol wise view in the side navbar.